examples/nextjs/src/components/Sidebar.tsx

```diff
@@ -89,10 +89,9 @@ export default function Sidebar({ children }: { children: React.ReactNode }) {
   return (
     <>
       <div className="h-screen w-full absolute">
-        <Transition.Root show={sidebarOpen} as={Fragment}>
```

Comment:
Curious what prompted this change.

Reply:
Some types changed, I don't know what the background is.

<img width="1249" alt="image" src="https://github.com/drifting-in-space/y-sweet/assets/46173/8ea496f2-e28a-4e1e-a895-826657be1fcb">

Link to full logs: https://vercel.com/drifting-corp/y-sweet-demos/9x7BCmmzLLQTazGuLwS8ohFeTqvh

Reply:
It makes the mobile menu look like this:

![Screenshot 2024-01-12 at 1 14 14 PM](https://github.com/drifting-in-space/y-sweet/assets/2099249/cff96932-9c0c-4d70-a987-9529927037e5)

But it should look like this:

![Screenshot 2024-01-12 at 1 14 07 PM](https://github.com/drifting-in-space/y-sweet/assets/2099249/923d63f6-3fb5-4049-9d88-6075fc08c7e4)

Reply:
Good catch. I'll see if there's anything I can do.